app/src/main/java/com/project/pv239/customtimealarm/Activities/SettingsActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.project.pv239.customtimealarm.Activities;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+
+import com.project.pv239.customtimealarm.Fragments.SettingsFragment;
+
+public class SettingsActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        if (savedInstanceState == null) {
```

Comment:
len moja otazka (neovaldam javu z pamati) - ide napisat !savedInstanceState? (namiesto blabla == null ze napises iba !blabla) :))

Reply:
podla mna to nejde a pride mi to takto aj citatelnejsie to == null:)